experimental/uefi/loader/Cargo.toml

```diff
@@ -21,4 +22,6 @@ tokio = { version = "*", features = [
   "process",
   "signal",
 ] }
-tokio-util = { version = "*", features = ["codec"] }
+tokio-serde-cbor = "*"
```

Comment:
This is fine for now, but there are potential incompatibilities between `ciborium` and `serde-cbor` if we start using more complex types. See https://docs.rs/ciborium/latest/ciborium/#compatibility-with-other-implementations

Reply:
Fair enough. In addition, `serde-cbor` is effectively unmaintained: https://crates.io/crates/serde_cbor

As we'll likely replace CBOR anyway I'll leave it as is for now.